src/dictionary/index.stories.mdx

```diff
@@ -0,0 +1,137 @@
+import { Meta, Source, Story } from "@storybook/addon-docs/blocks";
+import ColorSwatch, { ColorSwatchWrapper } from "util/storybook-docs/ColorSwatch";
+import { TableLayout } from 'util/storybook-docs/Table';
+import { mostReadable } from 'tinycolor2';
+import { colors, layouts, fonts, borders, motions } from '../../lib/dictionary/js/manifest.js';
```

Comment:
all code here is basically the same except for the HBS loops, which have been replaced by direct mappings over category properties from the manifest